Q/Qt/build_tarballs.jl

```diff
@@ -95,8 +98,23 @@ EOT
         export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
         export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
         export PKG_CONFIG_LIBDIR=$prefix/lib/pkgconfig
+
+        sed -i 's/^QMAKE_C.*FLAGS.*//' ../qt-everywhere-src-*/qtbase/mkspecs/devices/linux-rasp-pi3-g++/qmake.conf
         
         ../qt-everywhere-src-*/configure QMAKE_LFLAGS=-liconv -platform linux-g++ -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -extprefix $prefix $commonoptions \
+            -skip qtwinextras -fontconfig -sysroot /opt/$target/bin/../$target/sys-root
+        ;;
+        
+    *aarch64-linux*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
+        export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
+        export PKG_CONFIG_LIBDIR=$prefix/lib/pkgconfig
+
+        sed -i 's/^QMAKE_C.*FLAGS.*//' ../qt-everywhere-src-*/qtbase/mkspecs/devices/linux-rasp-pi4-v3d-g++/qmake.conf
```

Comment:
This file starts with
```
# qmake configuration for the Raspberry Pi 4 (32-bit) using the Mesa V3D
# graphics stack. (not the Broadcom stack)
```
doesn't sound right

Reply:
But the configuration doesn't look too wrong (apart from the hard-float thing which I'm not sure about)

Reply:
Can't you just use the generic linux configuration?

Reply:
Yes, the hard float thing is usually on armv7 and not on aarch64.

Reply:
Basically even though raspeberry pi 4 has an armv8 processor, it is a 32-bit OS, and you have to build everything as 32-bit- as if it is armv7 (but presumably some of the armv7 compiler flags don't work).

Reply:
Yep, switched to the generic version now, thanks for the suggestions!